ludwig/models/ecd.py

```diff
@@ -98,13 +99,27 @@ def save_savedmodel(self, save_path):
         keras_model = self.get_connected_model(training=False)
         keras_model.save(save_path)
 
-    def forward(self, inputs, training=None, mask=None):
-        # parameter inputs is a dict feature_name -> tensor / ndarray
-        # or
-        # parameter (inputs, targets) where
-        #   inputs is a dict feature_name -> tensor/ndarray
-        #   targets is dict feature_name -> tensor/ndarray
-
+    def forward(
+            self,
+            inputs: Union[
+                Dict[str, torch.Tensor],
```

Comment:
`inputs` can also be a Dict[str, numpy.ndarray]

Reply:
Thanks, updated!